gen/buf/validate/conformance/cases/bool_pb2.py

```diff
@@ -14,12 +14,23 @@
 
 # -*- coding: utf-8 -*-
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
+# NO CHECKED-IN PROTOBUF GENCODE
```

Comment:
?

Reply:
No idea.

https://github.com/protocolbuffers/protobuf/pull/16226

Probably used to prevent gencode in Piper, or something.